features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/Operations.jsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { withStyles, withTheme } from '@material-ui/core/styles';
+import { injectIntl } from 'react-intl';
+import Table from '@material-ui/core/Table';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+
+import PropTypes from 'prop-types';
+import Chip from '@material-ui/core/Chip';
+import Api from 'AppData/api';
+
+/**
+ *
+ *
+ * @param {*} props
+ * @returns
+ */
+function RenderMethodBase(props) {
+    const { theme, method } = props;
```

Comment:
theme and method is missing in props validation. Please check other Eslint issues and fix

Reply:
Fixed with 2a6fbdd3ac9ce54ca7bc94a45328a41826836a80